packages/web/src/components/Contributions/ContributionsBox.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Box, Link } from '@chakra-ui/react';
+
+export interface ContributionsBoxProps {
+  cbox: {
+    id: string;
+    nodeID: string;
+    type: string;
+    score: number;
+    contributedAt: Date;
+    description: string | null;
+    url: string;
+  };
+}
+
+export const ContributionsBox: React.FC<ContributionsBoxProps> = ({ cbox }) => {
+  return (
+    <Box maxW="sm" borderWidth="1px" borderRadius="lg" overflow="hidden" key={cbox.id}>
```

Comment:
Let's hop on a call and chat about this style stuff; I think there's likely a simpler way to do this using additional Chakra components